Assets/GothicVR/Scripts/UI/CloseMenueScript.cs

```diff
@@ -0,0 +1,23 @@
+using UnityEngine;
+
+public class CloseMenueScript : MonoBehaviour
+{
+    private Vector3 menuePosition;
+    private Quaternion menueRotation;
+    private GameObject menueParent;
+    private void Start()
+    {
+        menuePosition = gameObject.transform.localPosition;
+        menueRotation = gameObject.transform.localRotation;
+        menueParent = gameObject.transform.parent.gameObject;
+        gameObject.SetActive(false);
+    }
+
+    public void closeFunction()
+    {
+        gameObject.transform.parent = menueParent.transform;
```

Comment:
```gameObject``` is redundant here. In fact it's slightly more inperformant (but neglectable) as it just fetches the GO and then the Transform.
![image](https://github.com/GothicVRProject/GothicVR/assets/120568393/3dc0becc-9cdd-4470-aace-cc18a2ed9653)

Reply:
That was a remnant of an old implementation that was sometime like "button.gameobject.transform.."
Resolved.